src/c-writer.cc

```diff
@@ -3466,39 +4367,121 @@ void CWriter::Write(const SimdLaneOpExpr& expr) {
 }
 
 void CWriter::Write(const SimdLoadLaneExpr& expr) {
-  UNIMPLEMENTED("SIMD support");
+  const char* func = nullptr;
+  switch (expr.opcode) {
+    case Opcode::V128Load8Lane: func = "v128_load8_lane"; break;
+    case Opcode::V128Load16Lane: func = "v128_load16_lane"; break;
+    case Opcode::V128Load32Lane: func = "v128_load32_lane"; break;
+    case Opcode::V128Load64Lane: func = "v128_load64_lane"; break;
+    default:
+        printf("issue with SimdLoadLaneExpr opcode %s \n", expr.opcode.GetName());
+        WABT_UNREACHABLE;
+  }
+  Memory* memory = module_->memories[module_->GetMemoryIndex(expr.memidx)];
+  Type result_type = expr.opcode.GetResultType();
+  Write(StackVar(0, result_type), " = ", func, expr.val, "(", ExternalInstancePtr(memory->name), 
+        ", (u64)(", StackVar(1), ")");
+
+  if (expr.offset != 0)
+    Write(" + ", expr.offset, "u");
+  Write(");", Newline());
+
+  DropTypes(2);
+  PushType(result_type);
 }
 
 void CWriter::Write(const SimdStoreLaneExpr& expr) {
-  UNIMPLEMENTED("SIMD support");
+    const char* func = nullptr;
+  // wasm_v128_store16_lane(in_a, res, 2);
+  switch (expr.opcode) {
+  case Opcode::V128Store8Lane: func = "v128_store8_lane"; break;
+  case Opcode::V128Store16Lane: func = "v128_store16_lane"; break;
+  case Opcode::V128Store32Lane: func = "v128_store32_lane"; break;
+  case Opcode::V128Store64Lane: func = "v128_store64_lane"; break;
+  default:
+      printf("issue with SimdStoreLaneExpr opcode %s \n", expr.opcode.GetName());
+      WABT_UNREACHABLE;
```

Comment:
Indention looks odd here and in the first line of the function.  Maybe `git clang-format origin/main`?